app/legal/styles.css

```
@@ -1,179 +1,323 @@
+  margin: 0;
+  padding: 0;
 }
+
+/* Anchor tags (links) */
 a {
-	font-family: 'Figtree', sans-serif;
-	color: rgba(102, 102, 102, 1);
+  color: #007bff;
+  text-decoration: none;
+  font-family: var(--font-sans);
+  color: var(--gray-text);
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

color is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)